package.json

```diff
@@ -142,7 +142,9 @@
         "tar": "~6.1.11",
         "tcp-ping": "~0.1.1",
         "thirty-two": "~1.0.2",
+        "tippy.js": "^6.3.7",
```

Comment:
A bit more context is needed on this one:
We already have popperjs installed.
Is there a reason why you chose this library?

Reply:
Yes, the [Vue-Calendar-Heatmap](https://razorness.github.io/vue3-calendar-heatmap/) has suggested to use it as mentioned in the documentation.

Reply:
Can we use popperjs instead for these popups?

Reply:
@CommanderStorm Tippy.js seems to be a peer dependency to the calendar heat-map package. As per their [documentation](https://razorness.github.io/vue3-calendar-heatmap/) this is a required dev dependency. I also tried disabling tooltip and removing the tippy.js dependency, that throw an error `ERROR: Could not resolve "tippy.js"`. For now I have moved them to the dev dependency.